### ISSUE TITLE
Single IPP Ratio Measure numerator/denominator populations are not treated independently

### ISSUE THREAD

**Issue by mgramigna** [projecttacoma/fqm-execution#182]
Discussed in meeting with @srankins, @p9g, and others.

Per HQMF, ratio measure calculation should always treat numerator and denominator as independent populations. See below pseudo-cql from HQMF:

![image](https://user-images.githubusercontent.com/16297930/211086955-0fd373a9-ff07-4617-a5bd-c4b52146fe4b.png)

Some diagrams in the [cqfm IG](http://hl7.org/fhir/us/cqfmeasures/2023Jan/measure-conformance.html#ratio-measures) seem to be consistent with this, however as @p9g noted, the [pseudo-cql](http://hl7.org/fhir/us/cqfmeasures/2023Jan/measure-conformance.html#patient-based-calculation-1) does not align with HQMF.

Neither does fqm-execution, per the following example:

```cql
// snip
define "ipp":
  true

define "denom":
  false

define "numer":
  true
```

In the context of a ratio measure, "numer" should still be true despite "denom" being false. This is the current behavior in fqm-execution *only* when the measure has multiple IPPs, but fqm-execution should also behave this way even with a single IPP. Currently, it does not:

```json
[
  {
    "populationType": "initial-population",
    "criteriaExpression": "ipp",
    "result": true,
    "populationId": "ec813784-2eb4-46a9-b86a-bb03bf1c5b25"
  },
  {
    "populationType": "numerator",
    "criteriaExpression": "numer",
    "result": false,
    "populationId": "304711ea-6ded-4145-b429-20551850fc83"
  },
  {
    "populationType": "denominator",
    "criteriaExpression": "denom",
    "result": false,
    "populationId": "4c919f82-a385-4cda-9324-ae6c1722dec6"
  }
]
```

Measure Bundle and Patient Bundle for really simple demonstration of the issue: [issue.zip](https://github.com/projecttacoma/fqm-execution/files/10362543/issue.zip) (the above example)

[CMS871](https://github.com/cqframework/ecqm-content-r4-2021/tree/master/bundles/measure/HospitalHarmHyperglycemiainHospitalizedPatientsFHIR) is also an example of this issue in fqm-execution.

**Comment by p9g** [comment]
https://github.com/HL7/cqf-measures/blob/master/archived/old_images/RatioVenn.jpg
https://github.com/HL7/cqf-measures/blob/master/archived/old_images/Ratio2Venn.png
https://github.com/HL7/cqf-measures/blob/master/archived/old_images/Ratio3Venn.png

**Comment by p9g** [comment]
HQMF spec downloadable from http://www.hl7.org/implement/standards/product_brief.cfm?product_id=97

**Comment by mgramigna** [comment]
Fixed in [v1.0.5](https://github.com/projecttacoma/fqm-execution/releases/tag/v1.0.5)
